.github/ISSUE_TEMPLATE/new-member.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Organization Membership Request
+about: Request membership in Crossplane Org
+title: 'MEMBERSHIP: <your-GH-handle>'
+
+---
+
+### GitHub Username
+e.g. (at)example_user
+
+### Requirements
+- [ ] I have reviewed the [community membership guidelines](https://github.com/crossplane/crossplane/blob/master/GOVERNANCE.md#organization-members)
+- [ ] I have [enabled 2FA](https://github.com/settings/security) on my GitHub account
+- [ ] I have joined the [Crossplane slack channel](https://slack.crossplane.io/)
+- [ ] I am actively contributing to the Crossplane project
+- [ ] I have two sponsors that are current Crossplane Org members
```

Comment:
let's make sure these requirements are reflected in the governance as well: https://github.com/crossplane/crossplane/blob/master/GOVERNANCE.md#organization-members

can you make a corresponding update there? you could also consider updating that section simply to point/link to the resources in this repo, but it may be preferrable to have everything explicitly spelled out within the governance.